### PR TITLE
Bugfix/net2 ntp

### PR DESCRIPTION
**Pull request by k8soneill**
Changes /etc/chrony.conf on net2 servers to use custom sources

### CODE REVIEW

**Comment by stevemul** [comment]
The problem with using a template is that we assume we know best - there are quite a few values defined in the template that could be changed in the default config and we'd not get those changes. How much extra effort is involved in making this not depend on the template? For example could we define the sources in a different file? Or use a chrony command to apply the additional servers and make them persist?

**Comment by k8soneill** [comment]
Passed pipeline test

**Review by MudasarUKCloud — APPROVED** [review]
ok

File: roles/common/tasks/main.yml [hunk 1]
```diff
@@ -39,6 +39,28 @@
...
+  replace:
+    path: /etc/chrony.conf
+    regexp: '^server.*iburst$'
+    replace: ''
+  when: multinetwork and inventory_hostname in groups.net2_all
+
+- name: Add net2 ntp servers to chrony config file
+  lineinfile:
+    path: /etc/chrony.conf
+    regexp: '^#\sPlease\sconsider'
```
Comment (stevemul) [inline roles/common/tasks/main.yml]: Is this line needed? I didn't think you needed to match this line as well? Just use insertafter on its own.

**Review by stevemul — CHANGES_REQUESTED** [review]

File: roles/common/tasks/main.yml (hunk 1 above)
Comment (k8soneill) [reply roles/common/tasks/main.yml]: That line wasn't needed  - removed regexp line

**Review by stevemul — APPROVED** [review]
LGTM

**Pull request merged**